pom.xml

```diff
@@ -172,7 +172,7 @@
         <dependency>
           <groupId>com.floragunn</groupId>
           <artifactId>search-guard-static-tcnative-beta</artifactId>
-          <version>1.1.0j-${netty-native.version}-non-fedora-linux-x86_64</version>
```

Comment:
Not checking whether these files exist. Assuming you've checked this exists in both our local repo and com.floragunn.